exercises/sublist/description.md

```diff
@@ -1,18 +1,24 @@
 # Description
 
-Given two lists determine if the first list is contained within the second
-list, if the second list is contained within the first list, if both lists are
-contained within each other or if none of these are true.
+Given any two lists `A` and `B`, determine if:
 
-Specifically, a list A is a sublist of list B if by dropping 0 or more elements
-from the front of B and 0 or more elements from the back of B you get a list
-that's completely equal to A.
+* List `A` is equal to list `B`; or
+* List `A` contains list `B` (`A` is a superlist of `B`); or
+* List `B` contains list `A` (`A` is a sublist of `B`); or
+* None of the above is true, thus lists `A` and `B` are unequal
+
+Specifically, a list `A` is equal to list `B` if both contain only the same sequence of values.
```

Comment:
Do we need to specify that "sequence" means: the same values _in the same order_?

Reply:
What do you think, which version would work better?

1. ...both contain only the same values in the same order (i.e. they are the same exact sequence of values).
2. ...both are the same exact sequence of values, i.e. they only contain the same values in the same order.
3. ...both contain only the same values in the same order, i.e. the same sequence.

I would probably lean towards no. 2.

Reply:
2 looks good to me